NEWS.md

```diff
@@ -22,6 +22,8 @@
 
 * Calculation of step in slider for `RangeFilterState` now uses `checkmate::test_integerish` instead of `is.integer`.
 * Updated `init_filtered_data` to take into account the removal of `CDISCTealData` from `teal.data` package.
+* Parameters `input_dataname` and `output_dataname` of `FilterStates` are now combine into one character parameter `dataname`.
+* Parameter `varname` of `FilterState` is not a character.
```

Comment:
both classes and their constructor wrappers (init_filter_state, init_filter_states) are internal. They shouldn't be mentioned in the changelog

Reply:
There are also other NEWS about theese clases - we will remove them later